faunadb-android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="com.faunadb.client">
+<manifest package="com.faunadb.client"
+          xmlns:android="http://schemas.android.com/apk/res/android"
+          android:versionName="3.0.0-SNAPSHOT" >
+  <uses-sdk
+      android:minSdkVersion="26"
+      android:targetSdkVersion="26" />
```

Comment:
you can fix this by adding `minSdkVersion in Android := "26"` on `build.sbt`

Reply:
Aha! Thanks.